handler/dimensions.go

```diff
@@ -30,6 +31,78 @@ func NewDimensions(cfg *config.Config, r responder, c cantabularClient, d datase
 	}
 }
 
+// GetDimensionCategories is the handler for GET/population-types/{population-type}/dimension-categories?dims=xxx
+func (h *Dimensions) GetDimensionCategories(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	req := contract.GetDimensionCategoriesRequest{
+		PopulationType: chi.URLParam(r, "population-type"),
+	}
+
+	logData := log.Data{
+		"request": req,
+	}
+
+	if err := parseRequest(r, &req); err != nil {
+		h.respond.Error(ctx, w, http.StatusBadRequest, &Error{
+			err: errors.Wrap(err, "failed to get dimension categories"),
```

Comment:
Marked as resolved but didn't update or reply?

Reply:
Sorry that should be done now